sdcm/cluster_k8s/__init__.py

```diff
@@ -1970,9 +1970,15 @@ def node_setup(self, node: BaseScyllaPodContainer, verbose: bool = False, timeou
 
         if TestConfig.MULTI_REGION:
             node.datacenter_setup(self.datacenter)  # pylint: disable=no-member
+        try:
+            # NOTE: case of seedful scylla (operator v1.3.0-)
+            seed_nodes = ','.join(self.seed_nodes_ips)
+        except AssertionError:
+            # NOTE: case of seedless scylla (operator v1.4.0+)
+            seed_nodes = None
         self.node_config_setup(
             node,
-            seed_address=','.join(self.seed_nodes_ips),
+            seed_address=seed_nodes,
```

Comment:
I don't see that the seed_nodes is used in `sdcm.cluster_k8s.ScyllaPodCluster.node_config_setup`, any idea why?

Reply:
It is used in "is_seed" property. And this property is used for bunch of nemesis we run against K8S clusters.
So, this var is really needed, because it causes effect.